Content.Client/Chat/UI/EmotesMenu.xaml.cs

```diff
@@ -46,13 +48,29 @@ public EmotesMenu()
                 !speech.AllowedEmotes.Contains(emote.ID))
                 continue;
 
+            string name;
+
+            if (emote.Category == EmoteCategory.GlobalCustom)
+            {
+                var value = _cfg.GetCVar<string>(emote.ID).Trim();
+
+                if (string.IsNullOrEmpty(value))
+                    continue;
+
+                name = value;
+            }
+            else
+            {
+                name = emote.Name;
+            }
+
             var parent = FindControl<RadialContainer>(emote.Category.ToString());
 
             var button = new EmoteMenuButton
             {
                 StyleClasses = { "RadialMenuButton" },
                 SetSize = new Vector2(64f, 64f),
-                ToolTip = Loc.GetString(emote.Name),
+                ToolTip = Loc.GetString(name),
```

Comment:
You should not be running the custom emote text through `Loc.GetString()`.

Reply:
Yes, I think I was experimenting if I could resolve the pronouns and adjectives and then accidentally left it in the code.